packages/cli/src/cli/commands/sites.ts

```diff
@@ -16,6 +18,39 @@ export const createSite = errorHandler<string>(async (url: string) => {
   }
 });
 
+type getComponentSchemaParams = { url: string; apiPath: string | null };
+export const getComponentSchema = errorHandler<getComponentSchemaParams>(
+  async ({ url, apiPath }: getComponentSchemaParams) => {
+    const spinner = ora("Creating site...").start();
+    try {
+      const schemaEndpoint = `${url}${
+        apiPath || "/api/pantheoncloud/component_schema"
+      }`;
+      const result = (await axios.get(schemaEndpoint)).data;
+
+      spinner.succeed(
+        `Retrieved component schema from ${schemaEndpoint}. Now checking its validity`,
+      );
+
+      try {
+        validateComponentSchema(result);
+      } catch (e) {
```

Comment:
May be we can 
```
console.error(chalk.red(<message>))
process.exit(1)
```
Just to be consistent with other error message formatting.

Reply:
Do we need to call spinner.fail() too?